README.md

```diff
@@ -1,4 +1,35 @@
 # Monasca Helm
 
-Contains helm charts for our Monasca components as well as a top level Monasca helm chart that can be used to deploy
-Monasca.
\ No newline at end of file
+This repo contains Helm charts for Monasca and its dependencies. Each chart release is hosted on
+[monasca.io](http://monasca.io) via github pages.
+
+## Quick Start
+
+To install Monasca in Kubernetes you can follow the following steps:
+
+```bash
+$ helm repo add monasca http://monasca.io/monasca-helm-repo
+$ helm install monasca/monasca --name monasca --namespace monitoring
+```
+
+By default Monasca will monitor pod workloads, basic Kubernetes Health and autodetect Prometheus endpoints.
+
+### Accessing Data via Grafana
+
+Apart of the Monasca install is Grafana with default Kubernetes graphs. This can be accessed by port-forwarding the
+grafana service to localhost.
+
+Setting up grafana port-forward:
+```bash
+$ kubectl get pods -n monitoring -l component=grafana
+$ kubectl port-forward {{ grafana_pod_name_from_output_above }} -n monitoring 3000
+```
+
+After the above is set up you can visit [grafana](http://localhost:3000) with the default credentials mini-mon/password
+
+For more details on configuring the Monasca chart you can refer to the chart's [README](monasca/README.md) and for
+general details around Monasca in Kubernetes you can refer to [monasca.io Kubernetes](http://monasca.io/docs/kubernetes.html)
+
+## Local Development Environment
+
+For Local development we use [minikube](https://github.com/kubernetes/minikube) for deploying Monasca.
```

Comment:
I'd maybe recommend putting the minikube recommendation further up, probably as part of the quickstart